c_common/front_end_common_lib/src/malloc_extras.c

```diff
@@ -658,3 +658,34 @@ void *malloc_extras_malloc(uint bytes) {
     // if no safety, the point is the point used by the application code.
     return (void *) p;
 }
+
+void *malloc_extras_malloc_reverse(uint bytes) {
+    if (safety) {
+        bytes = bytes + EXTRA_BYTES;
+    }
+
+    // try DTCM if allowed (not safe if overused, due to stack overflows)
+    int *p = NULL;
+    p = safe_sdram_malloc(bytes);
```

Comment:
Comment "try DTCM" and  call "safe_sdram"... do not match up.

Reply:
fixed